docs/TALKS.md

```diff
@@ -14,7 +37,14 @@ In this impromptu talk, matklad discusses build and release process of TigerBeet
 
 https://www.youtube.com/watch?v=eFTQzhfO6nc
 
-## October 2, 2023: Redesigning OLTP for a New Order of Magnitude at QCon San Francisco 2024
+## 23 October, 2023: P99 CONF 2023 | A Deterministic Walk Down TigerBeetle’s main() Street
+
+Learn how to use Zig to implement a fully deterministic distributed system which will never fail with an out of memory error, for predictable performance and 700x faster tests, with matklad.
```

Comment:
@MarinaWoo let's hard-wrap the lines at about 100 characters to remain consistent with the rest of our docs here.

Hard-wrapping is useful because sometimes people look at the raw source without wrapping, like this

https://raw.githubusercontent.com/tigerbeetle/tigerbeetle/448bd60ff40bc1cad71633fdb0f01c73fcd573a7/docs/TALKS.md

In that context, long lines are poorly readable

Reply:
Okay on it.